code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -9,8 +9,8 @@
  * Glass sheets
  */
 GLOBAL_LIST_INIT(glass_recipes, list ( \
-	new/datum/stack_recipe("directional window", /obj/structure/window/unanchored, time = 0, on_floor = TRUE, window_checks = TRUE), \
-	new/datum/stack_recipe("fulltile window", /obj/structure/window/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE) \
+	new/datum/stack_recipe/window("directional window", /obj/structure/window, time = 0, on_floor = TRUE, window_checks = TRUE), \

```

Comment:
Okay apparently if you change the /obj/structure/window/Initialize(mapload, direct) to have a third parameter you can set the anchored to normally TRUE and override it here to FALSE. I presume you would need to setup a spawner for the corner of the tile windows to do this too and probably have a fourth parameter to set their dir too.

Secondly I was incorrect in regards to src with your datums that is apparently fine.